dali/kernels/slice/slice_gpu.cuh

```diff
@@ -36,78 +36,123 @@ template <int Dims>
 struct SliceSampleDesc {
   void *__restrict__ out;
   const void *__restrict__ in;
+
+  const void *__restrict__ fill_values;
+  int channel_dim;
+
   TensorShape<Dims> in_strides;
   TensorShape<Dims> out_strides;
+
+  TensorShape<Dims> anchor;
+  TensorShape<Dims> in_shape;
 };
 
 struct BlockDesc {
   int sampleIdx;
-  size_t offset;
-  size_t size;
+  int64_t offset;
```

Comment:
Maybe `ssize_t` instead?

Reply:
I think we agreed some time ago to use int64_t. I am just adjusting now because that decision came after this kernel was written